DirectProgramming/DPC++FPGA/Tutorials/Features/mem_channels/README.md

```diff
@@ -0,0 +1,306 @@
+Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide)
+is the reference manual for targeting FPGAs through DPC++. <br>
+The [oneAPI Programming
+Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general
+resource for target-independent DPC++ programming.
+
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04/20.04, RHEL*/CentOS* 8, SUSE* 15; Windows* 10
+| Hardware                          | Intel® Programmable Acceleration Card (PAC) with Intel Arria® 10 GX FPGA <br> Intel® FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix® 10 SX) <br> Intel® FPGA 3rd party / custom platforms with oneAPI support <br> *__Note__: Intel® FPGA PAC hardware is only compatible with Ubuntu 18.04* 
+| Software                          | Intel® oneAPI DPC++ Compiler <br> Intel® FPGA Add-On for oneAPI Base Toolkit 
+| What you will learn               | How and when to use the `mem_channel` buffer property and the `-Xsno-interleaving` flag
+| Time to complete                  | 30 minutes
+
+
+
+## Purpose
+
+This FPGA tutorial demonstrates an example of using the `mem_channel` buffer
+property in conjuction with the `-Xsno-interleaving` flag to reduce the amount
+of resources required to implement a DPC++ FPGA design.
+
+By default, the Intel® oneAPI DPC++ compiler configures each global memory type
+in a burst-interleaved manner where memory words are interleaved across the
+available memory channels. This usually leads to better throughput because it
+prevents load imbalance by ensuring that memory accesses do not favor one
+external memory channel over another. Moreover, burst-interleaving enables you
+to view a multi-channel memory as one large channel so you can write your code
+without worrying about where each buffer should be allocated. However, this
+configuration can be expensive in terms of FPGA resources because the global
+memory interconnect required to orchestrate the memory accesses across all the
+channels is complex. 
+
+The Intel® oneAPI DPC++ compiler allows you to avoid this area overhead by
+disabling burst-interleaving and assigning buffers to invidual channels. There
+are two advantages to such configuration:
+1. A simpler global memory interconnect is built which requires a smaller
+   amount of FPGA resources than the interconnect needed for the
+   burst-interleaving configuration.
+2. Potential improvements to the global memory bandwidth utilization due to
+   less contention at each memory channel.
+
+Burst-interleaving should only be disabled in situations where satisfactory
+load balancing can be achived by assigning buffers to individual channels.
+Otherwise, the global memory bandwidth utilization may be reduced which will
+negatively impact the throughput of your design. 
+
+To disable burst-interleaving, you need to assign a memory channel to each
```

Comment:
What does this mean? The act of using mem_channel disables burst-interleaving? Why do we need the flag then?

Maybe reword if that is not what you intended to communicate

Reply:
Addressed.